docs/installguide/system_requirements.rst

```diff
@@ -1,12 +1,87 @@
 System requirements
 ===================
 
+Operating systems
+-----------------
+
+ * Windows XP, Vista, 7, 8, 10
+ * Mac OSX 10.9, 10.10 and 10.11
+ * Linux: Any system with Python 2.7
+ * Debian/Raspberry Pi packages: Wheezy or later
+ * Ubuntu packages: 14.04, 15.10, 16.04 - anything that's *not* end-of-life.
+
+
 Supported Browsers
 ------------------
-KA Lite is currently *not* supported on Internet Explorer version 8 or lower. You must use IE9 or later.
 
-You can also use KA Lite on browsers like Firefox, Chrome, and Safari!
+ * IE9+
+ * Firefox
```

Comment:
But older versions of FF may be untested, but they are still supposed to work because of using jquery and old bootstrap! I think FF 45 would be too exclusive, but perhaps we can indicate that the most guaranteed version is the latest FF that we run our tests on...